plugins/discovery/discovery.go

```diff
@@ -38,7 +40,7 @@ type Discovery struct {
 	manager      *plugins.Manager
 	config       *Config
 	factories    map[string]plugins.Factory
-	downloader   *download.Downloader                // discovery bundle downloader
+	downloader   download.DownloaderInterface        // discovery bundle downloader
```

Comment:
We can probably use the [Loader](https://github.com/open-policy-agent/opa/blob/main/plugins/bundle/plugin.go#L44) interface here.